tools/tangle-cli/README.md

```diff
@@ -11,339 +11,549 @@ CLI for developing applications on the Tangle
 npm install -g @tangle-js/tangle-cli
 ```
 
-## Run 
+## Usage
 
-On the command line just run ```tcli```
-
-## Commands
+On the command line just run `tcli`.
 
 * [Decentralized Identities](#did)
 * [Verifiable Credentials](#verifiable-credentials-vc)
 * [Streams Channels](#channels) (Powered by IOTA Streams)
 * [Tangle Indexed Messages](#tangle-messages)
 
+> For all commands you must specify at minumum `--devnet`, `--mainnet` or `--net <netId> --node <nodeUrl>`
+
 ```
 tcli [command]
 
 Commands:
   tcli did      DID operations
-  tcli vc       VC  Operations
-  tcli channel  Streams Channels operations
+  tcli vc       Verifiable Credential operations
+  tcli channel  Streams Channels operations (Powered by IOTA Streams)
   tcli msg      Tangle message (indexation payloads) operations
 
 Options:
-  --version  Show version number                                       [boolean]
-  --mainnet  IOTA Mainnet                                              [boolean]
-  --devnet   IOTA devnet                                                [boolean]
-  --net, -n  Node's endpoint or other IOTA network                     [string]
-  --help     Show help                                                 [boolean]
+  --version        Show version number                                 [boolean]
+  --devnet         Default settings for IOTA Chrysalis Devnet
```

Comment:
instead of repeating this info across all the document it would be better to have it one time at the beginning of the README